src/20_Components/amp-access.html

```diff
@@ -0,0 +1,88 @@
+<!-- #### Introduction
+  The [amp-access](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md) adds support for paywalls and subscriptions to AMP, allowing the publisher to control which content is accessible by the reader and with what restrictions. 
```

Comment:
add component in "The [amp-access](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md) component adds"